.hooks/pre-push

```diff
@@ -1,9 +1,29 @@
 #!/bin/sh
+# exit on error
+set -e
 
-printf "\n⏳ composer lint\n" && \
-vendor/bin/php-cs-fixer fix --verbose --dry-run --show-progress=none && \
-printf "\n⏳ npm lint\n" && \
-npm run lint && \
-printf "\n⏳ composer analyse\n" && \
-vendor/bin/phpstan analyse --memory-limit 512M --no-progress && \
-printf "\n✅ pre-push hook\n\n"
+modifiedPhpFiles="$(git diff master...HEAD --diff-filter=d --name-only '**/*.php')"
+if [ ${#modifiedPhpFiles} -gt 0 ]; then
+    printf "\n⏳ composer analyse\n"
+    vendor/bin/phpstan analyse --memory-limit 512M --no-progress
+
+    printf "\n⏳ composer lint\n"
+    vendor/bin/php-cs-fixer fix --verbose --dry-run --show-progress=none --config=.php-cs-fixer.php ${modifiedPhpFiles}
+
+#    printf "\n⏳ composer test\n"
+#    phpunit
+fi
```

Comment:
Without the "&&" chains, I don't think this will stop when an error occurs. Maybe you should add an `|| exit 1` to each command?

Reply:
The `e` flag should take care of this:
```
# exit on error
set -e
```

> -e  Exit immediately if a command exits with a non-zero status.

It stopped for me when encountering errors.

Reply:
Ah, missed that. And/or didn't realize that's what it did. :)